### ISSUE TITLE
Delegations view on some accounts is broken

### ISSUE THREAD

**Issue by lukaw3d** [oasisprotocol/oasis-wallet-web#2030]
https://wallet.dev.oasis.io/account/oasis1qprtzrg97jk0wxnqkhxwyzy5qys47r7alvfl3fcg/stake/active-delegations
![image](https://github.com/user-attachments/assets/5c3b5a04-f560-4fd6-8714-d6c948bd3953)

**Comment by lukaw3d** [comment]
https://api.oasisscan.com/mainnet/chain/account/delegations?size=500&address=oasis1qprtzrg97jk0wxnqkhxwyzy5qys47r7alvfl3fcg
```json
...
      {
        "validatorAddress": "oasis1qqs5wnxvsk009swtt7ehm5fslxve96kczszwt47s",
        "validatorName": null,
        "icon": null,
        "entityAddress": null,
        "shares": "1549017.970963851",
        "amount": "2024435.120714304",
        "active": true
      },
      {
        "validatorAddress": "oasis1qpwrq93z8s9ytu2hfjtqggc9edgwfadzevs3trvm",
        "validatorName": "Munay Network",
        "icon": "https://s3.amazonaws.com/keybase_processed_uploads/e46b46540baf3a06ac7babf28a06db05_360_360.jpg",
        "entityAddress": null,
        "shares": "1281083.498020046",
        "amount": "1534115.76456032",
        "active": true
      },
      {
        "validatorAddress": null,
        "validatorName": null,
        "icon": null,
        "entityAddress": "oasis1qpz6gtyzec4dwq7vxhvseg9yutavyht36qjsyqm2",
        "shares": "1000",
        "amount": "1000",
        "active": false
      },
      {
        "validatorAddress": null,
        "validatorName": null,
        "icon": null,
        "entityAddress": "oasis1qrcc0c2tppgkc4ue55ykaxe2yznr7zlmkg7vw5a0",
        "shares": "1000",
        "amount": "1000",
        "active": false
      },
      {
        "validatorAddress": null,
        "validatorName": null,
        "icon": null,
        "entityAddress": "oasis1qrct7ry2dcmk4ng2lcskzdv26c09xxmm4ytrlzzd",
        "shares": "1000",
        "amount": "1000",
        "active": false
      },
      {
        "validatorAddress": null,
        "validatorName": null,
        "icon": null,
        "entityAddress": "oasis1qrqz3yx3ujewxpuunlhd43xx3dcxwt90rcajfh3p",
        "shares": "1000",
        "amount": "1000",
        "active": false
      },
      {
        "validatorAddress": "oasis1qpjuke27se2wnmvx6e8uc4l5h44yjp9h7g2clqfq",
        "validatorName": "RockX",
        "icon": "https://s3.amazonaws.com/keybase_processed_uploads/1f47c0873584c67d4692e7abcee2bf05_360_360.jpg",
        "entityAddress": null,
        "shares": "286.065838182",
        "amount": "382.405095629",
        "active": true
      },
      {
        "validatorAddress": "oasis1qrw82ag2sypeytse9x9k4uxym53l5lc5jyfs2sxv",
        "validatorName": null,
        "icon": null,
        "entityAddress": null,
        "shares": "8.382409009",
        "amount": "11.004382002",
        "active": true
      },
...
```

**Comment by lukaw3d** [comment]
Bookmarklet to see debonding UI
```js
javascript:/* use on https://wallet.oasis.io/ */;
reactContainerField = Object.keys(document.querySelector('#root')).find(field=>field.startsWith('__reactContainer'));
store = document.querySelector('#root')[reactContainerField].memoizedState.element.props.store;
function requireUnscrambled(moduleName) {;
  const parcelRequire = Object.entries(window).find(a=>a[0].startsWith('parcelRequire'))[1];
  const scrambledName = Object.values(parcelRequire.modules).flatMap(a=>Object.entries(a[1])).find(a=>a[0]===moduleName)[1];
  return parcelRequire(scrambledName);
};
;
;
store.dispatch({ type: 'network/selectNetwork', payload: 'mainnet'});
store.dispatch({type: 'wallet/walletOpened', payload: { type: 'ledger', address: 'oasis1qprtzrg97jk0wxnqkhxwyzy5qys47r7alvfl3fcg', publicKey: '14cdafbd7818c7427d9908a6762b87bf0bdbf737418625ebc831f8f92392b7c8', path: [44, 474, 0, 0, 0], pathDisplay: "m/44'/474'/0'/0'/0'", balance: {available:'100000000'}}});
store.dispatch({type: 'wallet/selectWallet', payload: 'oasis1qprtzrg97jk0wxnqkhxwyzy5qys47r7alvfl3fcg'});

```
